servlet/src/main/scala/org/http4s/servlet/DefaultFilter.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2013 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s.servlet
+
+import javax.servlet._
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+
+@deprecated("Not releated to http4s. Will be removed from public API", "0.23.17")
+trait DefaultFilter extends Filter {
```

Comment:
Oh, I led you astray here.  I thought this was new, not on the way out.  Yes, it should be removed.

Reply:
Removed. Up in the next PR